espeakng/__init__.py

```diff
@@ -20,38 +20,39 @@
 import logging
 import subprocess
 import tempfile
+import platform
 
 class ESpeakNG(object):
 
-    def __init__(self, 
-                 volume      =         100, 
+    def __init__(self,
+                 volume      =         100,
                  audio_dev   =        None,
                  word_gap    =          -1, # ms
                  capitals    =           0, # indicate capital letters with: 1=sound, 2=the word "capitals", higher values indicate a pitch increase (try -k20).
                  line_length =           0, # Line length. If not zero, consider lines less than this length as end-of-clause
                  pitch       =          50, # 0-99
-                 speed       =         175, # approx. words per minute 
-                 voice       = 'english-us'):
+                 speed       =         175, # approx. words per minute
+                 voice       = 'en'):
```

Comment:
there's no voice named 'english-us' by espeak-ng default installation, you can set it to 'en' or 'en-us'

Reply:
Please do not change indentation. If voice names have changed, I guess we need to update tests as well.

Reply:
there are no indentations changed, only some white space trimmed at line end

Reply:
same problem here. should switch to `en-use`